### ISSUE TITLE
addimaged likes to connect out to the internet for no apparent reason.

### ISSUE THREAD

**Issue by majestrate** [probonopd/go-appimage#135]
according to the source this program likes to connect out on plaintext to some server on aws.

https://github.com/probonopd/go-appimage/blob/d1d2094ae63ec632bb204c83be62b38af48fdbfd/internal/helpers/mqtt.go#L76

can this be hidden behind an opt in analytics build tag? or does this do something else entirely?

**Comment by probonopd** [comment]
This is an experimental feature to notify the user about available updates of AppImages.
It should probably be deactivated from builds that are used for anything but experimentation.

**Comment by majestrate** [comment]
i see, perhaps consider hiding such things behind a build tag, e.g.  `// +build experimental` so that if someone using production builds doesn't have potential bugs introduced by these experimental features.

**Comment by probonopd** [comment]
Indeed this would be an useful option. Do you know how to implement it?

**Comment by majestrate** [comment]
yes, using golang's build tags, see https://www.digitalocean.com/community/tutorials/customizing-go-binaries-with-build-tags

**Comment by dafi87** [comment]
On a machine that's not connected to the internet at all times, this produces a flood of log entries.

> Sep 05 09:15:24 xyz appimaged[2544]: 2022/09/05 09:15:24 MQTT client connected: false
> Sep 05 09:15:24 xyz appimaged[2544]: 2022/09/05 09:15:24 MQTT client connected: false
> Sep 05 09:13:24 xyz appimaged[2544]: 2022/09/05 09:13:24 MQTT client connected: false
> Sep 05 09:13:24 xyz appimaged[2544]: 2022/09/05 09:13:24 MQTT client connected: false
> Sep 05 09:11:24 xyz appimaged[2544]: 2022/09/05 09:11:24 MQTT client connected: false
> Sep 05 09:11:24 xyz appimaged[2544]: 2022/09/05 09:11:24 MQTT client connected: false
> Sep 05 09:09:24 xyz appimaged[2544]: 2022/09/05 09:09:24 MQTT client connected: false
> Sep 05 09:09:24 xyz appimaged[2544]: 2022/09/05 09:09:24 MQTT client connected: false
> Sep 05 09:07:24 xyz appimaged[2544]: 2022/09/05 09:07:24 MQTT client connected: false
> Sep 05 09:07:24 xyz appimaged[2544]: 2022/09/05 09:07:24 MQTT client connected: false
> Sep 05 09:05:24 xyz appimaged[2544]: 2022/09/05 09:05:24 MQTT client connected: false
> Sep 05 09:05:24 xyz appimaged[2544]: 2022/09/05 09:05:24 MQTT client connected: false
> Sep 05 09:03:24 xyz appimaged[2544]: 2022/09/05 09:03:24 MQTT client connected: false
> Sep 05 09:03:24 xyz appimaged[2544]: 2022/09/05 09:03:24 MQTT client connected: false
> Sep 05 09:01:24 xyz appimaged[2544]: 2022/09/05 09:01:24 MQTT client connected: false
> Sep 05 09:01:24 xyz appimaged[2544]: 2022/09/05 09:01:24 MQTT client connected: false
> Sep 05 08:59:24 xyz appimaged[2544]: 2022/09/05 08:59:24 MQTT client connected: false
> Sep 05 08:59:24 xyz appimaged[2544]: 2022/09/05 08:59:24 MQTT client connected: false
